docker/fedora-25/docker-entrypoint.sh

```diff
@@ -45,12 +46,14 @@ if [ $1 = "mysqld" ]; then
 	EOSQL
 
 	# install plugin and create default db
+	# TODO: Use --init-file https://mariadb.com/kb/en/library/server-system-variables/#init_file
 	mysql --protocol=socket -uroot <<-EOSQL
 		INSTALL PLUGIN pinba SONAME 'libpinba_engine2.so';
```

Comment:
`--init-file` is designed for his purpose. I will allow storing all related init queries in the dedicated `*.sql` file.